### PR TITLE
Add io.liri.Text

### PR DESCRIPTION
**Pull request by plfiorini**
Text editor with Material Design from Liri

https://liri.io
https://github.com/lirios/text

### CODE REVIEW

**Comment by plfiorini** [comment]
This depends on #654

File: io.liri.Text.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+app-id: io.liri.Text
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.11
```
Comment (TingPing) [inline io.liri.Text.yaml]: This is a string, put it in quotes.

Comment (plfiorini) [reply io.liri.Text.yaml]: Done, thanks.

**Comment by barthalion** [comment]
bot, build io.liri.Text

**Comment by barthalion** [comment]
bot, build io.liri.Text

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/io.liri.Text

Thanks!

**Comment by plfiorini** [comment]
Thanks

**Pull request closed without merging**